src/main/java/com/demoqa/NonStaticCinfigFileReader.java

```diff
@@ -0,0 +1,32 @@
+package com.demoqa;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Properties;
+
+public class NonStaticCinfigFileReader {
```

Comment:
```suggestion
public class NonStaticConfigFileReader {
```
Azirov in da house?)

Reply:
Done